### ISSUE TITLE
Apologies in advance...

### ISSUE THREAD

**Issue by davidjhawkins** [BenjaminWatts/gb-solar-storage#1]
Hi Ben
Apologies, I saw your X-post which referenced this repo and wanted to take a look from a different POV (I'm based in the electricity industry and dabble in R)

Here's some quick questions that may indicate how stupid I am with reading PY

1 - I've downloaded the weather_history.json file into R
It appears to comprise time series data in a raw format (e.g. -915152400000)
Maybe R has corrupted this but please would you indicate what these time values are?

2 - The resulting data table comprises 733,680 rows -- is this a year's worth of data?

3 - You create a load_profile
Is this the same profile for each and every day of the period/year?
I know that there are profiles around (I think Elexon has something somewhere) which may be better here

4 - You use Heat loss coefficient in KW/°C (default is 0.2 KW/°C)
Does this mean that for a dwelling to remain at 16C indoors while ambient is -10C the heat loss = 26 * 0.2 ?
e.g. heat loss at constant -10C ambient = 26 * 0.2 * 24 = 124kWh
and, I guess that with a COP of between 3.5 and 2.5 this indicates a ASHP demand of 35-50kWh (probably correct for a very cold day, not intended as a comment on thermal accuracy)

5 - Your thermal calcs reminds me of the degree-days / degree-hours seen elsewhere - happy to share background docs

Thanks for sharing your code via GH and apologies for any silly questions

With best regards
David Hawkins
